plugins/spamassassin.js

```diff
@@ -224,7 +225,7 @@ exports.score_too_high = function (conn, spamd_response) {
         return "spam score exceeded threshold";
     }
 
-    return false;
+    return null;
```

Comment:
Please explain why switching this from boolean to null is an improvement.

Reply:
@msimerson the error on these involved returning the same type. true/false is boolean, but the other returns were objects / structures.

Reply:
But this change fails that test as well. Switching to a null doesn't return the same type.

Reply:
![function returns](https://user-images.githubusercontent.com/1007551/209407710-568864d2-2880-433a-b9a1-a4ebca844180.png)

It'll even convert that last part into `return max && (score >= max) ? "spam score exceeded threshold" : null;` if so desired.

Reply:
Maybe return a '' string instead?

Reply:
Now we're talking. An empty string is still falsy and that would make the type consistent.

Reply:
Done! Made sure the test was also working with it.